python/cugraph/cugraph/testing/resultset.py

```diff
@@ -43,6 +43,7 @@ def get_cudf_dataframe(self):
 
 
 _resultsets = {}
+_results_dir_path = RAPIDS_DATASET_ROOT_DIR_PATH / "tests" / "resultsets"
```

Comment:
I noticed this constant is also defined in `generate_resultsets.py`.  Can we define this path in just one place and use it in both modules?  I think having it defined here in `resultset.py' might be better.

Also, I noticed this imports `RAPIDS_DATASET_ROOT_DIR` from `cugraph.testing`, and `generate_resultsets.py` imports it from `cugraph.testing.utils`, so I think we should be using the same import for that too.